ansi2html/converter.py

```diff
@@ -454,7 +454,11 @@ def _handle_ansi_code(
                 continue
 
             try:
-                params = list(map(int, params.split(";")))
+                params = [
+                    int(item)
+                    for subitem in [x.split(":") for x in params.split(";")]
+                    for item in subitem
+                ]
```

Comment:
That seems to be way more complex than ideal and necessary. Would this approach work?:

```
In [7]: [int(v) for v in re.split("[:;]", "111;222:333;444")]
Out[7]: [111, 222, 333, 444]
```

Reply:
Probably. I keep forgetting that `re.split()` exists.

Reply:
@jaysonlarose if you could simplify the code using `re.split`, I would consider it ready to merge. Thank you!

Reply:
Done!